libs/slam/async_slam/async_slam.cpp

```
@@ -193,6 +193,11 @@ void AsyncSlam::TrackResult(FrameId frameId, int64_t timestamp_ns, const odom::I
     }
     if (tail_.UpdateTimeByOdometry(timestamp_ns, current_pose)) {
       input_queue_.Push(vo_keyframe);
+      const size_t queued_commands = input_queue_.Size();
```

Clang (14.0.6)

[warning] variable 'queued_commands' is not initialized

(cppcoreguidelines-init-variables)